README.md

```diff
@@ -1 +1,411 @@
+---
+apiVersion: azure.crossplane.io/v1beta1
+kind: ProviderConfig
+metadata:
+  name: default
+  namespace: upbound-system
+spec:
+  credentials:
+    source: Secret
+    secretRef:
+      namespace: upbound-system
+      name: azure-account-creds
+      key: credentials
+EOF
+```
+
+Apply it to the cluster:
+```console
+kubectl apply -f "provider.yaml"
+```
+The output will look like the following:
+
+```shell
+provider.azure.crossplane.io/default created
+```
+
+Crossplane resources use the ProviderConfig named ```default``` if no specific ProviderConfig is specified, so this ProviderConfig will be the default for all Azure resources.
+
+### We are now ready to provision resources:
+
+#### Create Network Fabric
+
+The example network composition includes the creation of a Resource Group, Virtual Network, and Subnets:
+
+```console
+kubectl apply -f examples/network.yaml
+```
+
+Verify status:
+
+```console
+kubectl get claim
+kubectl get composite
+kubectl get managed
+```
+
+#### Invite App Teams to you Organization in Upbound Cloud
+
+1. Create a Team `team1`.
+1. Invite app team members and grant access to `Control Planes` and `Repositories`.
+
+App Dev/Ops: Consume the infrastructure you need using kubectl
+
+#### Join your Organization in Upbound Cloud
+
+1. **Join** your [Upbound Cloud](https://cloud.upbound.io/register)
+   `Organization`
+1. Verify access to your team `Control Planes` and Registries
+
+#### Provision a CompositePostgreSQLInstance in your team Control Plane GUI console
+
+1. Browse the available self-service APIs (XRDs) Control Plane
+1. Provision a `CompositePostgreSQLInstance` using the custom generated GUI for your
+Platform `Configuration`
+1. View status / details in your `Control Plane` GUI console
+
+#### Connect kubectl to your team Control Plane
+
+1. Connect `kubectl` to a `Control Plane` from the self-service GUI console.
+
+#### Provision a PostgreSQLInstance using kubectl
+
+```console
+kubectl apply -f examples/postgres-claim.yaml
+```
+
+Verify status:
+
+```console
+kubectl get claim
+kubectl get composite
+kubectl get managed
+```
+
+### Cleanup & Uninstall
+
+#### Cleanup Resources
+
+Delete resources created through the `Control Plane` Configurations menu:
+
+* From the `Teams` GUI using the ellipsis menu in the resource view.
+* Using `kubectl delete -n team1 <claim-name>`.
+
+Delete resources created using `kubectl`:
+
+```console
+kubectl delete -f examples/postgres-claim.yaml
+kubectl delete -f examples/network.yaml
+```
+
+Verify all underlying resources have been cleanly deleted:
+
+```console
+kubectl get managed
+```
+
+#### Uninstall Provider & Platform Configuration
+
+```console
+kubectl delete configurations.pkg.crossplane.io platform-ref-azure
+kubectl delete providers.pkg.crossplane.io provider-azure
+kubectl delete providers.pkg.crossplane.io provider-helm
+```
+
+#### Uninstall Crossplane kubectl plugin
+
+```console
+rm /usr/local/bin/kubectl-crossplane*
+```
+
```

Comment:
Suggestion: 

#### Uninstall Azure App Registration

First get the 

```console
az ad sp list --display-name platform-ref-azure
```

Get the value from the field `appId` and use it to delete the Service Principal:

```console
az ad sp delete --id <value of appId>"
```